custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -532,7 +549,41 @@ def updates(self):
                 sum(CASE WHEN age_tranche in ('0','6','12','24') THEN wer_eligible ELSE 0 END) AS wer_eligible_0_2,
                 sum(CASE WHEN age_tranche in ('0','6','12','24') THEN nutrition_status_weighed ELSE 0 END) AS wer_weighed_0_2,
                 sum(thr_eligible) as thr_eligible,
-                sum(rations_21_plus_distributed) as rations_21_plus_distributed
+                sum(rations_21_plus_distributed) as rations_21_plus_distributed,
+                sum(CASE WHEN gender='F' AND age_tranche::INTEGER<=36 THEN
```

Comment:
All this data is already in agg child health, can you not just use that table for the API? This table is massive already and continuing to add more columns makes it very hard to use.